crates/context_aware_config/src/helpers.rs

```diff
@@ -197,6 +207,43 @@ pub fn calculate_context_priority(
     }
 }
 
+fn calculate_2_exponenet_to_bigdecimal(exponent: u32) -> Result<BigDecimal, String> {
```

Comment:
Why use bigdecimal instead of big int?

Reply:
bigint wont support value larger than 2^64